src/infrastructure/schemas/user-saloon.schema.ts

```diff
@@ -0,0 +1,25 @@
+import { Column, Entity, JoinColumn, ManyToOne, OneToOne, PrimaryGeneratedColumn } from 'typeorm';
+import { User } from './user.schema';
+import { Saloon } from './saloon.schema';
+import { SaloonRoles } from '@domain/enums/saloon-roles.enum';
+
+@Entity({ name: 'User_Saloon' })
+export class User_Saloon {
+  @PrimaryGeneratedColumn()
+  id: number;
+
+  @ManyToOne(() => User, (user) => user.saloons)
+  @JoinColumn({ name: 'userId' }) // Specify the foreign key column
```

Comment:
```suggestion
  @JoinColumn({ name: 'user_id' }) // Specify the foreign key column
```
https://brainstation.io/learn/sql/naming-conventions#:~:text=Though%20these%20vary%20somewhat%20between,names%20should%20not%20contain%20spaces.

Reply:
use a SnakeNamingStrategy
https://stackoverflow.com/questions/52931663/how-can-i-specify-column-name-casing-in-typeorm-migrations